Ntust_Cloud_System/Ntust_Cloud_System/FileOperator/FileOperator.h

```diff
@@ -0,0 +1,30 @@
+//
+//  FileOperator.h
+//  Ntust_Cloud_System
+//
+//  Created by 林仁鴻 on 2021/4/13.
+//
+
+
+#ifndef FileOperator_h
+#define FileOperator_h
+
+#include "ResponseHandler.h"
+#include <string>
+using namespace std;
```

Comment:
Please do not use "using namespace std;", it would pollute the file's namespace. See:
https://google.github.io/styleguide/cppguide.html#Namespaces

Please reference the class/symbol under the namespace directly, i.e. std::string
Or, place the using directive within the class.